src/parse/stmt.rs

```diff
@@ -278,6 +284,7 @@ impl<I: Lexer> Parser<I> {
     /// for_statement:
     ///     FOR '(' expr_opt ';' expr_opt ';' expr_opt ') statement
     ///   | FOR '(' declaration expr_opt ';' expr_opt ') statement
```

Comment:
```suggestion
    /// ```yacc
    /// for_statement:
    ///     FOR '(' expr_opt ';' expr_opt ';' expr_opt ') statement
    ///   | FOR '(' declaration expr_opt ';' expr_opt ') statement;
    /// ```
```